CHANGELOG.md

```diff
@@ -0,0 +1,353 @@
+# Changelog
+
+## 0.9.0 (2023-12-10)
+
```

Comment:
We could go with something like "first release" or leave this blank and remove everything below.

Reply:
Don't like that huge, mostly meaningless, changelog either. Would need to look into how to stop it from being generated, this seems to be good option https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#starting-commit.

Reply:
We could sneak that little change in with next merge to avoid polluting history. 🤷

Reply:
Nice, sounds like that starting commit would keep release please happy.

Let's do this!

Reply:
https://github.com/nvim-tree/nvim-tree.lua/pull/2588 that might be all we need.

Reply:
Yup, let's do this.

Worst case? We have a large changelog that we clean later.

Reply:
It failed https://github.com/nvim-tree/nvim-tree.lua/actions/runs/7161359020/job/19496856718#step:2:44.
But once we have user facing commit on master it might do what we want it to, so would be good to get something with feat or fix next.

Reply:
#2571 merged... let's see...

Reply:
This branch updated following CI "success": https://github.com/nvim-tree/nvim-tree.lua/actions/runs/7257707903/job/19771925430

It missed the tag bit - is there something we need to do?

Reply:
Let's try putting `Release-As: 0.9.0` in body of some user facing PR (and squashed commit message(!)) at which point we will like to cut that release.